cephprimarystorage/cephprimarystorage/cephagent.py

```
@@ -1251,6 +1247,50 @@ def replace_luks_volume(self, req):
+            raise Exception('failed to remove RBD image during LUKS rollback: %s' % path)
+
+    @replyerror
+    def rollback_luks_volume_conversion(self, req):
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
+        rsp = AgentResponse()
+        try:
+            self.rollback_luks_volume_conversion_path(
+                self._normalize_install_path(cmd.installPath),
+                self._normalize_install_path(cmd.targetInstallPath),
+                self._normalize_install_path(cmd.sourceTrashInstallPath))
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)